packs/slack/sensors/slack_sensor.yaml

```diff
@@ -20,3 +20,17 @@
             type: "integer"
           timestamp_raw:
             type: "string"
+    -
+      name: "message_channel_join"
```

Comment:
My thinking is that it's a subtype. What I really wanted to do was namespace it as such (`message.channel_join`), but I really was unsure what the effects would be. So, just went with the underscores.

Think I could get away with the namespace?